lib/koans/07_pattern_matching.ex

```diff
@@ -99,4 +110,9 @@ defmodule PatternMatching do
     %Animal{name: name} = %Animal{kind: "dog", name: "Max"}
     assert name == ___
   end
+
+  koan "Structs will even match with a regular map" do
+    %{name: name} = %Animal{kind: "dog", name: "Max"}
+    assert name == ___
```

Comment:
Trying to get them thinking "huh, structs really are a lot like maps" which sets us up to illustrate exactly how they're _not_ later.